python/pyspark/sql/types.py

```diff
@@ -1549,7 +1549,7 @@ def __repr__(self):
             return "Row(%s)" % ", ".join("%s=%r" % (k, v)
                                          for k, v in zip(self.__fields__, tuple(self)))
         else:
-            return "<Row(%s)>" % ", ".join(self)
+            return "<Row(%s)>" % ", ".join("%s" % (fields) for fields in self)
```

Comment:
https://github.com/apache/spark/pull/20503#discussion_r225846669

Reply:
I applied the suggested change.